[Stream] Unsubscription consequences - PSPEC-658 (#2641)
cloudflare/cloudflare-docs: products/stream/src/content/faq/index.md

</TableWrap>

### If I cancel my stream subscription, are the videos deleted?

> Videos are removed If the subscription is not renewed within 30 days.

Reviewer: ```suggestion
Videos are removed if the subscription is not renewed within 30 days.
```

Author: Thank you!

### How does Stream Live handle RTMP reconnections?

> As long as your streaming software reconnects, Stream Live will continue to ingest and stream your live video. Make sure the streaming software you are using to push RTMP feeds automatically reconnects if the connection breaks. Some apps like OBS reconnect automatically. Other apps like FFmpeg require custom configuration.

Reviewer: ```suggestion
As long as your streaming software reconnects, Stream Live will continue to ingest and stream your live video. Make sure the streaming software you use to push RTMP feeds automatically reconnects if the connection breaks. Some apps like OBS reconnect automatically while other apps like FFmpeg require custom configuration.
```

Author: Those weren't by me, not sure what marked that as diff?  and thanks for catching it :)
